shopinvader_api_signin/routers/signin.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, Response, status
+
+from odoo.addons.base.models.res_partner import Partner
+from odoo.addons.fastapi_auth_jwt.dependencies import (
```

Comment:
this module should be probably named `shopinvader_api_signin_jwt`
But in fact.... why is it bound to shopinvader at all?
Looks like a generic api endpoint, no?

Reply:
> this module should be probably named shopinvader_api_signin_jwt

+1

> But in fact.... why is it bound to shopinvader at all?

It will likely include logic to convert an anonymous partner into a real partner, recover the cart created while anonymous and such things.